protocol/contracts/libraries/LibUnripe.sol

```diff
@@ -235,6 +239,6 @@ library LibUnripe {
         uint256 supply
     ) internal view returns (uint256 redeem) {
         AppStorage storage s = LibAppStorage.diamondStorage();
```

Comment:
Could we change the name of the function to something more representive of the logic? Previously this got the % underlying share but now this includes the penalty logic.